modules/enabling-multi-cluster-console.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * assemblies/web-console.adoc
+
+:_content-type: PROCEDURE
+[id="enable-multi-cluster-console_{context}"]
+= Enabling multi-cluster in the web console
+
+:FeatureName: Multi-cluster console
+include::snippets/technology-preview.adoc[leveloffset=+1]
+//
+
+.Prerequisites
+* You must have link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html/install/index[Red Hat Advanced Cluster Management(ACM) for Kubernetes 2.5] installed.
+
+* You must have upgraded to the latest version of {product-title}.
+
+.Procedure
+
+. Log in to the {product-title} web console using your login credentials.
+
+. Enable the feature gate from *Cluster Settings* -> *Configuration* -> *FeatureGate*, and click on *enable*.
```

Comment:
@opayne1 I would add a warning here saying that this should not be set on production clusters. You will not be able to upgrade your cluster after applying the feature gate, and it can be undone.

Reply:
The steps are a little different. Currently you have to edit the YAML. It's currently documented here:

https://docs.openshift.com/container-platform/4.9/nodes/clusters/nodes-cluster-enabling-features.html